ntbs-service/Pages/Notifications/Overview.cshtml.cs

```diff
@@ -15,41 +15,33 @@ namespace ntbs_service.Pages_Notifications
 {
     public class OverviewModel : NotificationModelBase
     {
+        public OverviewModel(INotificationService service) : base(service) {}
 
-        public OverviewModel(INotificationService service) : base(service)
-        {
-            this.service = service;
-        }
-
-        public override async Task<IActionResult> OnGetAsync(int id, bool isBeingSubmitted)
+        public async Task<IActionResult> OnGetAsync(int id)
         {
             Notification = await service.GetNotificationWithAllInfoAsync(id);
             if (Notification == null)
             {
                 return NotFound();
             }
 
+            await GetLinkedNotifications();
             NotificationId = Notification.NotificationId;
 
-            if (Notification.NotificationStatus == NotificationStatus.Draft) {
+            if (Notification.NotificationStatus == NotificationStatus.Draft)
+            {
                 return RedirectToPage("./Edit/Patient", new {id = NotificationId});
             }
 
             return Page();
         }
 
-        protected override IActionResult RedirectToNextPage(int? notificationId)
+        public async Task<IActionResult> OnPostCreateLinkAsync()
         {
-            // This is not needed on the overview page. We should think about restructuring the
-            // inheritance to accommodate pages like this one without this hack

```

Comment:
Now we're not inheriting from NotificationEditModelBase, we don't need these NotImplementedExceptions :)